### DISCUSSION TITLE
Mixed operation of configuration storage settings-store and symfony-config

### DISCUSSION THREAD

**Discussion by podarcis** [pimcore/pimcore#13645] [Q&A / Support] [answered]
Let's say there are two Datahub configurations:

1. GraphQL config - which is maintained by developers and is under version control and thus part of a deployment.
2. Data Importer config - which is maintained by the customer himself (`prod` environment).

Pimcore's default configuration storage is `symfony-config` and that isn't writable in the `prod` environment, [as documented](https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html#page_Configuration-Storage-Locations-Fallbacks). The configuration storage `settings-store` on the other hand is located in the database and writeable in the `prod` environment.

Now my question: Is it "supported" to use `symfony-config` for Datahub configs in the `dev` environment (maintaining the GraphQL config) and use `settings-store` in the `prod` environment (the Data Importer config maintained by the customer; of course not under version control).

```
# .env.local on Env:dev
PIMCORE_WRITE_TARGET_DATA_HUB=symfony-config    # default
```
```
# .env.local on Env:prod
PIMCORE_WRITE_TARGET_DATA_HUB=settings-store
```

For my understanding it should work. On `prod` environment the GraphQL config is shown and "Save" action is disabled (as this comes from `symfony-config`) and the Data Importer config is writable. However, I don't find anything regarding this "mixed operation mode" in the docs. What do you say?

**Comment by fashxp** [comment, marked as answer]
Yes it is.
